### PR TITLE
Add Environment variables

### PR DESCRIPTION
**Pull request by octo-gone**

### CODE REVIEW

File: cli/src/commands/squid/release.ts [hunk 1]
```diff
@@ -62,11 +80,12 @@ export default class Release extends Command {
                   }`;
         }
         this.log(`🦑 Releasing the squid at ${deployUrl}`);
-        const result = await release(
```
Comment (mo4islona) [inline cli/src/commands/squid/release.ts:65]: Delete an old function please

File: cli/src/commands/squid/release.ts [hunk 2]
```diff
@@ -49,6 +69,22 @@ export default class Release extends Command {
...
         );
+
+        const envs: Record<string, string> = {} 
+        
+        flags.env?.forEach((e: string)=>{
+            const v = getEnv(e);
+            envs[v.name] = v.value;
+        });
+
+        if (flags.envFile != undefined && existsSync(flags.envFile)) {
```
Comment (mo4islona) [inline cli/src/commands/squid/release.ts:80]: [DRY
](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Make a function to read file and parse

Comment (octo-gone) [reply cli/src/commands/squid/release.ts]: probably done not the best variant but safe

**Review by mo4islona — APPROVED** [review]
LGTM

**Pull request merged**